bindings/c/include/opendal.h

```diff
@@ -77,7 +77,10 @@ typedef enum opendal_code {
  * BlockingLister is designed to list entries at given path in a blocking
  * manner.
  *
- * Users can construct Lister by `blocking_lister`.
+ * Users can construct Lister by [`BlockingOperator::lister`] or [`BlockingOperator::lister_with`].
+ *
+ * - Lister implements `Iterator<Item = Result<Entry>>`.
```

Comment:
cc @Ji-Xinyou, it seems not a good idea to use rust comments as C binding's comment directly.

Reply:
True. I will raise an issue about this and look into this later on. Thanks for mention.